### ISSUE TITLE
correlation and mcsolve tests error without cython

### ISSUE THREAD

**Issue by ajgpitch** [qutip/qutip#449]
Some of the test have decorators like this:
```
@unittest.skipIf(_version2int(Cython.__version__) < _version2int('0.14') or
                 Cython_found == 0, 'Cython not found or version too low.')
```
with the Cython_found variable being set at the top of the file like this:
```
# find Cython if it exists
try:
    import Cython
except:
    Cython_found = 0
else:
    Cython_found = 1
```
The condition in the decorator will error, as the Cython version is checked even though it failed to import.

**Comment by kafischer** [comment]
Hi Alex, I fixed this bug in my latest PR #509.

Kevin

**Comment by nonhermitian** [comment]
This is closed in #509.
